app/Model/Organisation.php

```diff
@@ -456,4 +454,41 @@ public function checkDesiredOrg($suggestedOrg, $registration)
         }
         return $suggestedOrg;
     }
+
+    private function getCountryGalaxyCluster()
+    {
+        static $list;
+        if (!$list) {
+            $file = new File(APP . '/files/misp-galaxy/clusters/country.json');
```

Comment:
Add a check if file doesn't exists? It can happens if submodules are not updated

Reply:
Fixed, thanks!